But there is a red dot:
https://ps2.cd.percona.com/job/percona-server-5.7-pipeline/73/consoleText
-- Looking for include files valgrind/memcheck.h, valgrind/valgrind.h
-- Looking for include files valgrind/memcheck.h, valgrind/valgrind.h - not found
CMake Error at configure.cmake:662 (MESSAGE):
  Unable to find Valgrind header files
  valgrind/memcheck.h;valgrind/valgrind.h.  Make sure you have them in your
  include path.

--- a/docker/install-deps
+++ b/docker/install-deps
@@ -34,7 +34,7 @@ if [ -f /usr/bin/yum ]; then
         perl-JSON MySQL-python perl-Digest perl-Digest-MD5 perl-Digest-Perl-MD5 \
         numactl-devel git which make rpm-build ccache libtool redhat-lsb-core sudo libasan \
     "
-    if [[ "$(rpm --eval %rhel)" = "6" ]] && [[ "$(rpm --eval %_arch)" = "x86_64" ]]; then
+    if [[ "$(rpm --eval %rhel)" = "6" ]]; then
         cat <<-EOF | tee /etc/yum.repos.d/devtoolset-2.repo
 		[devtoolset-2]
 		name=devtoolset-2

--- a/jenkins/valgrind-param.yml
+++ b/jenkins/valgrind-param.yml
@@ -160,7 +160,15 @@
          type: user-defined
          name: DOCKER_OS
          values:
+          - centos:6
+          - centos:7
+          - i386/centos:6
+          - ubuntu:trusty
+          - ubuntu:xenial
           - ubuntu:artful
+          - ubuntu:bionic
+          - debian:jessie
+          - debian:stretch
     builders:
     - trigger-builds:
       - project: percona-server-5.7-pipeline

